src/pygenstability/plotting.py

```diff
@@ -420,9 +447,12 @@ def _plot_optimal_scales(all_results, ax, scales, ax1, ax2):
         ax2.axvline(scale, ls="--", color="C4")
 
 
-def plot_scan_plt(all_results, scale_axis=True, figure_name="scan_results.svg"):
+def plot_scan_plt(
+    all_results, figsize=(6, 5), scale_axis=True, figure_name="scan_results.svg"
+):
     """Plot results of pygenstability with matplotlib."""
     scales = _get_scales(all_results, scale_axis=scale_axis)
+    plt.figure(figsize=figsize)
```

Comment:
I think I didn't put this here, so one can call figure before this function, and have more freedom